### PR TITLE
Update draft-tigress-requirements.md

### PR DESCRIPTION
**Pull request by bleventh**

### CODE REVIEW

File: draft-tigress-requirements.md [hunk 1]
```diff
@@ -99,25 +99,25 @@ informative:
...
+The companies that are providing the digital credential for consumption by a digital wallet are the provisioning partners. They are in control of the provisioning information and the lifecycle of the credentials. Each digital wallet has a preexisting trust relationship between itself and the Provisioning Partner.
 
-The interface between the devices and the Provisioning Partner can be proprietary or part of published specifications such as the {{CCC-Digital-Key-30}}. The sender device obtains provisioning information from the provisioning partner, then shares it to the recipient device via Tigress. The recipient then takes that data and sends it to the Provisioning Partner to redeem a credential.
+The interface between the devices and the Provisioning Partner can be proprietary or part of a public standard such as the CCC. The sender obtains provisioning information from the provisioning partner, then shares it to the recipient via Tigress. The recipient then takes that data and sends it to the Provisioning Partner to redeem a credential for consumption in a digital wallet.
```
Comment (castiz) [inline draft-tigress-requirements.md:118]: this undid the CCC piece that Dmitry just edited. Have to add that back and make it "published specification".

File: draft-tigress-requirements.md [hunk 2]
```diff
@@ -99,25 +99,25 @@ informative:
...
 
-For some credential types the Provisioning Partner who mints new credentials is actually the sender device. In that scenario the receiver will generate new key material at the request of the sender device, and then communicate with the sender device over Tigress to its key material signed by the sender device.
+For some credential types the Provisioning Partner who mints new credentials is actually the sender. In that scenario the receiver will generate a new key material at the request of the sender, and then communicate with the sender over Tigress to have it's key material signed by the sender.
```
Comment (castiz) [inline draft-tigress-requirements.md:120]: its

File: draft-tigress-requirements.md [hunk 3]
```diff
@@ -132,12 +132,13 @@ General terms:
...
+- Let's say Ben owns a vehicle that supports digital keys which comply with the CCC {{CCC-Digital-Key-30}} open standard. Ben would like to let Ryan borrow the car for the weekend. Ryan and Ben are using two different mobile phones with different operating systems. In order for Ben to share his digital car key to Ryan for a weekend, he must transfer some data to the receiver device. The data structure shared between the two participants is defined in the CCC. In addition, the CCC requires the receiver to generate required key material and return it to the sender to sign and return back to the receiver. At this point, the receiver now has a token that will allow them to provision their new key with the car.
```
Comment (castiz) [inline draft-tigress-requirements.md:139]: re-add Dmitry's changes

**Comment by alexpell00** [comment]
I think these changes are overriding some other changes. You might need to rebase and update the PR.

**Pull request merged**